run_forward_server_test.sh

```diff
@@ -0,0 +1,17 @@
+killall forward_server
```

Comment:
(lint) Tips depend on target shell and yours is unknown. Add a shebang. [SC2148]